src/planner/plan/plan_read.cpp

```diff
@@ -146,6 +146,18 @@ void Planner::planGDSCall(const BoundReadingClause& readingClause,
             planReadOp(getGDSCall(readingClause), predicatesToPush, *plan);
         }
     }
+    if (bindData->hasNodeOutput()) {
```

Comment:
What's this change about? Is it connected to multi-label graphs or something independent?

Reply:
I'll soon open a PR to make algorithm not only returns node_id but also node properties.